docs/conf.py

```diff
@@ -43,11 +43,21 @@
 copyright = '2019, Qiskit Development Team'  # pylint: disable=redefined-builtin
 author = 'Qiskit Development Team'
 
+import qiskit_sphinx_theme
+
 # The short X.Y version
 version = ''
 # The full version, including alpha/beta/rc tags
 release = '0.12.0'
 
+rst_prolog = """
+.. |version| replace:: {0}
+""".format(release)
+
+html_static_path = ['_static']
```

Comment:
The only thing in `_static` is the `style.css` file which shouldn't be necessary anymore so we can remove this and the `_static` directory.

```suggestion
```